renderdoc/driver/vulkan/vk_core.h

```diff
@@ -1119,6 +1119,12 @@ class WrappedVulkan : public IFrameCapturer
 
   IMPLEMENT_FUNCTION_SERIALISED(VkResult, vkSetEvent, VkDevice device, VkEvent event);
 
+  IMPLEMENT_FUNCTION_SERIALISED(void, vkSetHdrMetadataEXT, VkDevice device, uint32_t swapchainCount,
+                                const VkSwapchainKHR *pSwapchains, const VkHdrMetadataEXT *pMetadata);
+
+  IMPLEMENT_FUNCTION_SERIALISED(void, vkSetLocalDimmingAMD, VkDevice device,
+                                VkSwapchainKHR swapChain, VkBool32 localDimmingEnable);
```

Comment:
Functions for extensions shouldn't be added in the middle here, instead add them to the end of the header with a comment for the extension they came from.